runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1961,9 +1961,28 @@ ZZ 31 bit zOS. See definition of J9TR_CAA_SAVE_OFFSET
 
 })dnl
 
-LOAD_ADDR_FROM_TOC(r14,TR_instanceOf)
+ifdef({J9ZOS390},{dnl
+
```

Comment:
Do we need to `RestoreSSP` to ensure the system stack pointer is restored before making the C call?

Reply:
Oh, I guess the code to restore SSP is already expanded: https://github.com/eclipse-openj9/openj9/pull/14443/files#diff-6870186d0528e479ad4329dd84386da041f665c28e65843be2f05f9406b8e5a1L1952-L1960

Reply:
@Spencer-Comin : Can I ask you to see if we can update the code here to use RestoreSSP and SaveSSP for better consistency?